jenkinsapi_tests/unittests/test_job.py

```diff
@@ -71,6 +89,18 @@ def test_name(job):
     assert job.name == 'foo'
 
 
+def test_get_job_url_diff_baseurl(job_with_diff_baseurl):
+    expected = "http://localhostdiff"
+    url = job_with_diff_baseurl.url
+    assert url == expected
+
+
+def test_get_job_url_same_baseurl(job_with_same_baseurl):
```

Comment:
Please don't put Jenkins initialization into fixture - this makes harder to read the code. Especially having a fixture that is used in single test!